urts/sys/types/src/lib.rs

```diff
@@ -7,3 +7,12 @@ use mc_sgx_core_sys_types::{
     sgx_attributes_t, sgx_config_id_t, sgx_config_svn_t, sgx_misc_select_t,
 };
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+
+impl Default for sgx_kss_config_t {
+    fn default() -> sgx_kss_config_t {
+        sgx_kss_config_t {
+            config_id: [0; 64],
```

Comment:
:wrench: Looks like I used a `64` or 2 in `mc-sgx-core-sys-types` :(
Although the rust compiler should bark if that `64` ever changes
```suggestion
    sgx_attributes_t, sgx_config_id_t, sgx_config_svn_t, sgx_misc_select_t, SGX_CONFIGID_SIZE,
};
include!(concat!(env!("OUT_DIR"), "/bindings.rs"));

impl Default for sgx_kss_config_t {
    fn default() -> sgx_kss_config_t {
        sgx_kss_config_t {
            config_id: [0; SGX_CONFIGID_SIZE],
```